vocode/streaming/transcriber/assembly_ai_transcriber.py

```diff
@@ -32,8 +32,8 @@ def __init__(
             )
         self._ended = False
         self.logger = logger or logging.getLogger(__name__)
-        if self.transcriber_config.endpointing_config:
-            raise Exception("Assembly AI endpointing config not supported yet")
+        # if self.transcriber_config.endpointing_config:
```

Comment:
we need to keep this code block! in case someone tries to use the endpointing config with AssemblyAI